gym/envs/tests/generate_json.py

```diff
@@ -0,0 +1,102 @@
+import gym
+from gym import envs, spaces 
+import json
+import numpy as np
+import datetime
+import os
+import hashlib
+
+import logging
+logger = logging.getLogger(__name__)
+
+from test_envs import should_skip_env_spec_for_tests
+
```

Comment:
Sounds good, I'll move it to scripts/generate_json.py. Is it reasonable to add an __init__.py to the envs/tests folder, since this script requires imports from there? http://stackoverflow.com/questions/4383571/importing-files-from-different-folder-in-python